js/resolc/package.json

```diff
@@ -32,6 +32,6 @@
         "@types/node": "^22.9.0",
         "commander": "^13.1.0",
         "package-json": "^10.0.1",
-        "solc": "^0.8.29"
+        "solc": "<=0.8.30"
```

Comment:
Shouldn't we also require a minimum version here?

Reply:
Yes

Reply:
Restricted it further. Good catch thanks!